falkordb_bulk_loader/label.py

```
@@ -63,50 +63,52 @@ def update_node_dictionary(self, identifier):
-                    self.query_buffer.send_buffer()
-                    self.reset_partial_binary()
-                    # Push the label onto the query buffer again, as there are more entities to process.
-                    self.query_buffer.labels.append(self.to_binary())
+                    try:
+                        row_binary = self.pack_props(row)
+                    except SchemaError as e:
+                        # TODO why is line_num off by one?
+                        raise SchemaError(
+                            "%s:%d %s"
+                            % (self.infile.name, self.reader.line_num - 1, str(e))
+                        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)